docs/vgclog_balancedexamples.md

```diff
@@ -0,0 +1,643 @@
+<mem></mem>                         (status of different types of memory)
+
+</mem-info>         
+
+</gc-end>
+
+<cycle-end>                               (partial cycle ends)
+
+<exclusive-end>                           (STW pause ends)
+```
+
+When the `balanced` partial GC cycle is triggered, the GC runs an STW pause. Application threads are halted to give the garbage collector exclusive access to the heap. The STW pause is recorded in the logs by the `<exclusive-start>` element. 
+
+```xml
+<exclusive-start id="184" timestamp="2021-02-26T11:11:42.310" intervalms="3745.790">
+  <response-info timems="3.138" idlems="1.056" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+```
+
+A `balanced` partial GC cycle is triggered when the region count for the *eden* space reaches a *taxation* threshold. At this threshold, the GC 'taxes' the application threads, who have been 'paid' with memory allocation, to run some GC work - in this case, a partial GC cycle.The logs record this trigger reason by using the`<allocation-taxation>` element.
+
+```xml
+<allocation-taxation id="185" taxation-threshold="2147483648" timestamp="2021-02-26T11:11:42.311" intervalms="3745.785" />
+```
+
+Details about the start of the cycle are recorded by the `<cycle-start>` element. The cycle is recorded as a `partial gc` with an `id=336`. Any subsequent elements associated with this cycle have a `contextid=186` to match the cycle's `id`. You can use this `contextid` value to distinguish the partial GC cycle increment and operations from interleaving increments and operations from other `balanced` cycles, such as global *mark* cycles.
+
+```xml
+<cycle-start id="186" type="partial gc" contextid="0" timestamp="2021-02-26T11:11:42.311" intervalms="3745.805" />
+
+```
+
+The partial cycle begins its one and only GC increment, recorded using the `<gc-start>` element. You can understand the effect that the increment operations have on the heap by comparing snapshots of the memory taken at the start and the end of the increment.  The child elements `<mem-info>` and `<mem>` of the `<gc-start>` and `<gc-end>` elements record the amount of memory available and where it is located in the heap.
+
+```xml
+<gc-start id="187" type="partial gc" contextid="186" timestamp="2021-02-26T11:11:42.311">
+  <mem-info id="188" free="897581056" total="4294967296" percent="20">
+    <mem type="eden" free="0" total="2147483648" percent="0" />
+    <arraylet-primitive objects="1" leaves="4" largest="4" />
+    <remembered-set count="2749664" freebytes="160705664" totalbytes="171704320" percent="93" regionsoverflowed="1" regionsstable="12" regionsrebuilding="0"/>
+  </mem-info>
+</gc-start>
+```
+
+As expected, at the start of this increment, the eden region is full. 856 MB (897581056 B) of the total 2048 MB (2147483648 B) *eden* space is available as free memory. 856MB (897,581,056 B) of the total 4096MB (4294,967,296 B) heap is available as free memory.
```

Comment:
.... eden area or space or regions (not just single region) is full.

middle sentence is not correct: acutally 0 bytes of eden is available